addon/bootstrap.js

```diff
@@ -52,10 +52,7 @@ this.Bootstrap = {
     // Check if the user is eligible to run this study using the |isEligible|
     // function when the study is initialized (install or upgrade, the latter
     // being interpreted as a new install).
-    if (
-      reason === this.REASONS.ADDON_INSTALL ||
-      reason === this.REASONS.ADDON_UPGRADE
-    ) {
+    if (reason === this.REASONS.ADDON_INSTALL) {
```

Comment:
Reverting this change from PR #93 per this [ongoing discussion](https://github.com/mozilla/shield-studies-addon-template/pull/49#discussion_r168928701).